### PR TITLE
[CONFORMANCE] Update Conformance ReadMe according new features

### PR DESCRIPTION
**Pull request by iefode**
### Details:
 - *Update Conformance ReadMe according new features*

### Tickets:
 - *ticket-id*

### CODE REVIEW

**Comment by iefode** [comment]
@p-durandin @alvoron @eshoguli Please take a look as users who tested the approach

File: src/tests/functional/plugin/conformance/test_runner/README.md [hunk 1]
```diff
@@ -1,35 +1,56 @@
...
+* `ReadIR_compareWithRefs` set allows to read IRs from folders recursive, infer it and compare plugin results with reference.
+* `OpImplCheckTest` set checks an operation plugin implementation status using simple syntetic single operation graph (`Implemented`/`Not implemented`). The suite checks only `compile_model` without comparation with the reference.  
+
+A result of `conformanceTests` run is `report_opset.xml` file. It demonstrates tests statistic like pass rate, passed, crashed, skipped failed tests and plugin implementation per operation for devices.
 
 ## How to build
 Run the following command in build directory:
 1. Generate CMake project:
    ```
    cmake -DENABLE_FUNCTIONAL_TESTS=ON ..
```
Comment (nkogteva) [inline src/tests/functional/plugin/conformance/test_runner/README.md:33]:
```suggestion
   cmake -DENABLE_TESTS=ON -DENABLE_FUNCTIONAL_TESTS=ON ..
```

**Review by sbalandi — APPROVED** [review]

**Review by alvoron — APPROVED** [review]

File: src/tests/functional/plugin/conformance/test_runner/README.md [hunk 2]
```diff
@@ -1,35 +1,56 @@
...
+   make --jobs=$(nproc --all) conformanceTests
+   make --jobs=$(nproc --all) apiConformanceTests
+   ```
+3. Build plugins to validate
+   ```
+   make --jobs=$(nproc --all) lib_plugin_name
    ```
    
-## How to generate Conformance operation set
+## How to generate Conformance IRs set
```
[the remark this answers is not in the document]
Comment (iefode) [reply src/tests/functional/plugin/conformance/test_runner/README.md:46]: check the final part of the doc :)

File: src/tests/functional/plugin/conformance/test_runner/README.md [hunk 3]
```diff
@@ -1,35 +1,56 @@
...
+The other part of API conformance suite is QueryModel validation:
+* `ReadIR_queryModel` tests validates `query_model` API using simple single operation graph (Conformance IR) based on model parameters.
+* `OpImplCheck` tests are simple syntetic checks to `query_model` and set implementation status per an operation.
+
+A result of `apiConformanceTests` run is 2 xml files: `report_api.xml` and `report_opset.xml`. The first one shows OpenVINO API entities test statistic (like passed/failed/creashed/skipped/hanged test number, passrates and implementation status) per OPenVINO API entity. The second one demonstrates `query_model` results per operation.
+
+
+
+### `Opset Conformance`
+The suite validates OpenVINO an operation plugin implemenatation using simple signgle operation graphs (Conformance IR) taken from models. The plugin inference output is compared with reference.
```
Comment (sbalandi) [inline src/tests/functional/plugin/conformance/test_runner/README.md:21]: signgle  -> single

**Review by sbalandi — APPROVED** [review]

File: src/tests/functional/plugin/conformance/test_runner/README.md [hunk 4]
```diff
@@ -89,4 +104,53 @@ python3 summarize.py --xml /opt/repo/infrastructure-master/thirdparty/gtest-para
...
+The script has the following srguments:
+* `-h, --help`          show this help message and exit
+* `-m MODELS_PATH, --models_path MODELS_PATH`
+                        Path to directory/ies contains models to dump subgraph (default way is download OMZ). If `--s=0` specify Conformance IRs directory
+* `-d DEVICE, --device DEVICE`
+                        Specify target device. Default value is CPU
+* `-ov OV_PATH, --ov_path OV_PATH`
+                        OV binary files path. The default way is try to find installed OV by `INTEL_OPENVINO_DIR` in environmet variables or to find the absolute path of OV repo (by using script path)
+* `-w WORKING_DIR, --working_dir WORKING_DIR`
+                        Specify working directory to save all artifacts as reports, model, conformance_irs and etc.
```
Comment (tsavina) [inline src/tests/functional/plugin/conformance/test_runner/README.md:130]:
```suggestion
                        Specify a working directory to save all artifacts, such as reports, models, conformance_irs, etc.
```

File: src/tests/functional/plugin/conformance/test_runner/README.md [hunk 5]
```diff
@@ -89,4 +104,53 @@ python3 summarize.py --xml /opt/repo/infrastructure-master/thirdparty/gtest-para
...
+* `-t TYPE, --type TYPE`
+                        Specify conformance type: `OP` or `API`. Default value is `OP`
```
Comment (tsavina) [inline src/tests/functional/plugin/conformance/test_runner/README.md:132]:
```suggestion
                        Specify conformance type: `OP` or `API`. The default value is `OP`
```

File: src/tests/functional/plugin/conformance/test_runner/README.md [hunk 6]
```diff
@@ -89,4 +104,53 @@ python3 summarize.py --xml /opt/repo/infrastructure-master/thirdparty/gtest-para
...
+* `-s DUMP_CONFORMANCE, --dump_conformance DUMP_CONFORMANCE`
+                        Set '1' if you want to create Conformance IRs from custom models/Downloaded models. In other case set `0`. Default value is '1'
```
Comment (tsavina) [inline src/tests/functional/plugin/conformance/test_runner/README.md:134]:
```suggestion
                        Set '1' if you want to create Conformance IRs from custom/downloaded models. In other cases, set `0`. The default value is '1'
```

File: src/tests/functional/plugin/conformance/test_runner/README.md [hunk 7]
```diff
@@ -1,35 +1,56 @@
...
 ## Description
-Conformance suit is a set of tests with parameters independent from plug-in specific and limitations. It contains:
-* `ReadIR`. Allow to read IRs from folders recursive, infer it and compare results with reference.
-* `OpImplCheckTest`. Allow to check operation plugin implementation status (`Implemented`/`Not implemented`).
+Conformance suites certifies plugin functionality using a set of tests with plugin specificity independent parameters. There are 2 types of conformance validation.
 
-  **NOTE**: This test suite is in active development. The implementation status based on all other test results.
-* 
+### `API Conformance`
+The suite checks OpenVINO API the folowing entities a plugin implementation (Also there are test instantiations to validate hardware plugin functionality via software plugins (e.g. `MULTI`, `HETERO` and etc.) for the entities):
```
Comment (tsavina) [inline src/tests/functional/plugin/conformance/test_runner/README.md:7]: The suite checks the following OpenVINO API entities in a plugin implementation:
		* plugin
		* compiled model (executable network)
		* infer request
Also, there are test instantiations to validate hardware plugin functionality via software plugins (e.g. `MULTI`, `HETERO` and etc.) for the entities.

File: src/tests/functional/plugin/conformance/test_runner/README.md [hunk 8]
```diff
@@ -89,4 +104,52 @@ python3 summarize.py --xml /opt/repo/infrastructure-master/thirdparty/gtest-para
...
+There is a simple python runner to complete the whole conformance pipeline locally. Some steps could be excluded from the pipeline by command-line parameter configuration.
+
+## The conformance pipeline steps:
+1. (Optional) Download OMZ models.
+2. (Optional) Convert models to IR or prepare a folder with IRs.
+3. (Optional) Run `SubgraphDumper` to generate a simple single op graph based on models or download the `conformance_ir` folder.
+4. Run conformance test executable files.
+5. Generate conformance reports.
+
+## Command line arguments
```
Comment (tsavina) [inline src/tests/functional/plugin/conformance/test_runner/README.md:119]: Command-line arguments

File: src/tests/functional/plugin/conformance/test_runner/README.md [hunk 9]
```diff
@@ -1,35 +1,57 @@
...
+
+A result of the `apiConformanceTests` run is two xml files: `report_api.xml` and `report_opset.xml`. The first one shows OpenVINO API entities' test statistics for each OpenVINO API entity, such as passed/failed/crashed/skipped/hanging, tests number, pass rates, and implementation status. The second one demonstrates the `query_model` results for each operation.
+
+
+
+### `Opset Conformance`
+The suite validates an OpenVINO operation plugin implementation, using simple single operation graphs (Conformance IR) taken from models. The plugin inference output is compared with the reference.
+
+ The suite contains:
+* `ReadIR_compareWithRefs` set allows reading IRs from folders recursively, infer them and compare plugin results with the reference.
```
Comment (tsavina) [inline src/tests/functional/plugin/conformance/test_runner/README.md:25]:
```suggestion
* `ReadIR_compareWithRefs` set allows reading IRs from folders recursively, inferring them, and comparing plugin results with the reference.
```

File: src/tests/functional/plugin/conformance/test_runner/README.md [hunk 10]
```diff
@@ -1,35 +1,57 @@
 # Conformance test runner
 
 ## Description
-Conformance suit is a set of tests with parameters independent from plug-in specific and limitations. It contains:
-* `ReadIR`. Allow to read IRs from folders recursive, infer it and compare results with reference.
-* `OpImplCheckTest`. Allow to check operation plugin implementation status (`Implemented`/`Not implemented`).
+Conformance suites certifies plugin functionality using a set of tests with plugin specificity independent parameters. There are 2 types of conformance validation.
```
Comment (tsavina) [inline src/tests/functional/plugin/conformance/test_runner/README.md:4]:
```suggestion
Conformance suites certify plugin functionality using a set of tests with plugin specificity independent parameters. There are two types of conformance validation.
```

**Review by tsavina — APPROVED** [review]

**Pull request merged**